cysec-platform-core/src/main/java/eu/smesec/cysec/platform/core/endpoints/Coaches.java

```diff
@@ -240,6 +241,8 @@ public Response updateAnswer(
         return Response.status(404).build();
       }
       CoachLibrary library = cal.getLibrariesForQuestionnaire(coachId).get(0);
+      
+      value = StringEscapeUtils.escapeHtml4(value);      
```

Comment:
@martingwerder as discussed: directly escape user text before passing to other components.
Downside: Assumption is made by escaping html which will work for html and probably xml but might not for others and a new [dependency ](https://commons.apache.org/proper/commons-text/)was introduced.